.gitignore

```diff
@@ -150,6 +150,11 @@ client/styleguide
 # webpack stats
 client/webpack-stats.json
 
+# Projects
+projects/*/build
+projects/*/dist
+projects/*/*.egg-info
+
```

Comment:
Dropping by to post-merge bikeshed... these should be `packages` now?

Reply:
https://github.com/galaxyproject/galaxy/pull/7656